icekit/workflow/admin.py

```diff
@@ -0,0 +1,76 @@
+from django.contrib import admin
+from django.contrib.auth import get_user_model
+from django.contrib.contenttypes.models import ContentType
+from django.contrib.contenttypes.admin import GenericTabularInline
+from django import forms
+
+from . import models
+
+
+class WorkflowStepForm(forms.ModelForm):
+
+    def __init__(self, *args, **kwargs):
+        super(WorkflowStepForm, self).__init__(*args, **kwargs)
+        # Limit user choices for `assigned_to` field to active staff users
+        User = get_user_model()
+        staff_user_pks = [
+            u.pk for u in User.objects.filter(is_active=True, is_staff=True)]
+        self.fields['assigned_to'].choices = \
+            [choice for choice in self.fields['assigned_to'].choices
+             if not choice[0] or choice[0] in staff_user_pks]
+
+
+class WorkflowStepTabularInline(GenericTabularInline):
+    model = models.WorkflowStep
+    form = WorkflowStepForm
+
+    # Permit only a sinlge workflow step relationships now for simplicity
+    extra = 0
+    min_num = 1
+    max_num = 1
+    can_delete = False
+
+
+class WorkflowMixinAdmin(admin.ModelAdmin):
+    list_display = (
+        "created_by_column", "last_edited_by_column", "workflow_steps_column")
```

Comment:
I will leave the `created_by_column` in place for now just in case ACMI do really want it but we can scrap it later if it's unnecessary